garrysmod/resource/localization/en/main_menu.properties

```diff
@@ -37,7 +37,7 @@ limit_physgun=Limit the Physics Gun
 
 persistent_mode_menu=Persistence file\: (empty=disabled)
 persistent_mode=Persistence file\:
-persistent_mode.help=If not empty, enables 'Make Persistent' option when you right click on props while holding C, allowing you to save them across sessions. Changing the filename will save existing persistent props, cleanup the whole map and load props from given file if it exists.
+persistent_mode.help=If not empty, enables 'Make Persistent' option when you right click on props while holding %+menu_context%, allowing you to save them across sessions. Changing the filename will save existing persistent props, cleanup the whole map and load props from given file if it exists.
```

Comment:
![image](https://github.com/user-attachments/assets/0bad4a13-df38-4eec-a104-935eaa8112e6)
Sadly, this is not as simple as you think. You will have to add some code to make it work.

Reply:
Oh I see I will change this tomorrow.